src/unix/fs.c

```diff
@@ -49,6 +49,7 @@
     defined(__FreeBSD__)    ||                                            \
     defined(__OpenBSD__)    ||                                            \
     defined(__NetBSD__)
+# include <sys/syslimits.h> /* PATH_MAX, IOV_MAX */
```

Comment:
There was no failure compiling on osx without this include. PATH_MAX is
defined in limits.h on osx AFAIK, but I would appreciate verification on
that